build.sbt

```diff
@@ -31,7 +31,7 @@ ivyXML :=
 libraryDependencies ++= Seq(
   "commons-io" % "commons-io" % "2.16.1",
   "commons-lang" % "commons-lang" % "2.6",
-  "org.apache.commons" % "commons-vfs2" % "2.9.0",
+  "org.apache.commons" % "commons-vfs2" % "2.9.0" exclude("org.apache.hadoop", "hadoop-hdfs-client"),
```

Comment:
this can be removed in future once version 2.10 is released https://github.com/apache/commons-vfs/pull/291